src/drivers/SqliteDriver.ts

```diff
@@ -1,28 +1,27 @@
 import { IDriver } from "./IDriver";
-import type { Database } from "better-sqlite3";
+import { Database } from "better-sqlite3";
 
 export class SqliteDriver implements IDriver {
-    private static instance: SqliteDriver;
-    database: Database;
+    private static instance: SqliteDriver | null = null;
+    private readonly database: Database;
 
-    constructor(path: string) {
-        // eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
This eslint disable error line is needed because otherwise eslint takes it as an error that the line under is requiring a file not in import statement section